### PR TITLE
Create wacky.json

### PR DESCRIPTION
**Pull request by Arzejames**

### CODE REVIEW

File: sub-logs/wacky.json
Comment (DEV-DIBSTER) [inline sub-logs/wacky.json:12]: Unable to view site as it's blocked by Google for phishing.

Comment (DEV-DIBSTER) [reply sub-logs/wacky.json:12]: Please provide screenshots.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: sub-logs/wacky.json
Comment (Arzejames) [reply sub-logs/wacky.json:12]: image

**Comment by clarkedev** [comment]
@Arzejames Please resolve the security issue from Google. We will not merge this domain request if Google is marking the site as dangerous.

**Comment by clarkedev** [comment]
This pull request will be closed as you have not responded @Arzejames. If you resolve this issue please recreate the pull request.

**Pull request closed without merging**
